frontiers/location_in_the_neocortex_a_theory_of_sensorimotor_object_recognition_using_cortical_grid_cells/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2018, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+FROM python:2.7.14
+
+# Install dependencies
+COPY requirements.txt /home/
+RUN pip install -r /home/requirements.txt
+
+# Copy context into container file system
+COPY . /home/
+WORKDIR /home/
+
```

Comment:
Add `MPLBACKEND` environment variable to avoid `DISPLAY` errors when generating the plots with `matplotlib`.

```diff
+ # Use matplotib 'Agg' backend
+ ENV MPLBACKEND "Agg"
```